tests/ThrottlingMiddlewareTest.php

```diff
@@ -21,8 +22,41 @@
  */
 class ThrottlingMiddlewareTest extends AbstractPackageTestCase
 {
+    /**
+     * Middleware to test.
+     *
+     * @var ThrottlingMiddleware
+     */
+    protected $middleware;
+
+    public function setUp()
+    {
+        parent::setUp();
+
+        // Set up a fake route.
+        Route::get('/dummy', ['middleware' => ThrottlingMiddleware::class, function () {
```

Comment:
Does it work if you put this stuff in a route annotated with `@before` in the phpdoc, to avoid having to override setup?